app/src/main/java/com/bogdan/codeforceswatcher/feature/users/redux/Sorting.kt

```diff
@@ -0,0 +1,21 @@
+package com.bogdan.codeforceswatcher.feature.users.redux
+
+import com.bogdan.codeforceswatcher.model.User
+import com.bogdan.codeforceswatcher.room.DatabaseClient
+
+object Sorting {
+
+    fun sort(users: List<User>, sortType: UsersState.SortType): List<User> =
+        when (sortType) {
+            UsersState.SortType.DEFAULT -> DatabaseClient.userDao.getAll().reversed()
```

Comment:
It's not sorting. I would say it should return just `users`, which should be `DatabaseClient.userDao.getAll().reversed()` when `sort` called.